frontend/test/app/components/ExperimentDisableDialog-test.js

```diff
@@ -27,8 +28,8 @@ describe('app/components/ExperimentDisableDialog', () => {
 
   it('should render a modal container', () => {
     expect(subject.find('.modal-container')).to.have.property('length', 1);
-    expect(subject.find('.modal-header').props()['data-l10n-args'])
-      .to.equal(JSON.stringify({ title: experiment.title }));
+    expect(findLocalizedById(subject, 'feedbackUninstallTitle').props()['$title'])
```

Comment:
I'm also kind of wondering if someday there's a way we can detect strings that have changed without a corresponding change in l10n ID. Way beyond scope for this PR, but might be part of what bugs me about using l10n IDs as hooks in tests. That is, changing the IDs breaks tests, but changing strings doesn't set any alarms off